packages/ui-components/src/components/tab-navigation/tab-navigation.types.ts

```diff
@@ -0,0 +1,11 @@
+export interface ITabNavigationItem {
+	tabKey: number | string;
+	label: string;
+	disabled?: boolean;
+	notificationColor?: string;
```

Comment:
I think this info should be inside the `ITabsNotificationDict` because is related. 

Suggestion: 
```
export interface ITabNavigationItem {
	tabKey: number | string;
	label: string;
	disabled?: boolean;
}

export interface ITabNotification {
	active: boolean;
	color?: string;
}
export interface ITabsNotificationDict {
	[tabKey: string]: ITabNotification;
	[tabKey: number]: ITabNotification;
}
```